.github/workflows/ci.yml

```diff
@@ -40,7 +40,7 @@ jobs:
       - name: Accessibility Tests
         run: npm run test:axe
       - name: Unit Tests
-        run: npx web-test-runner --config web-test-runner.config.js --group default --playwright --browsers chromium firefox webkit
+        run: npm run test:headless
```

Comment:
Isn't this going to change it from running in all browsers to running only in Chrome?

Reply:
Nope. The `vdiff` group explicitly passes `['chromium']` but the `unit` group doesn't pass anything so it'll default to all `ALLOWED_BROWSERS`

Reply:
Gotcha. So I guess a decision for us around whether we actually want that by default locally vs. how it's behaving now (Chrome only).

Reply:
Hmm, so you're thinking maybe we want this to be:
```
web-test-runner --group unit
```
And in `package.json`, we'd have `"test:headless": "web-test-runner --group unit --browsers chromium",`?

Reply:
I feel like defaulting to more will make sure people don't accidentally not test everything, but we can flesh this out more later.

Reply:
That would keep the behaviour the same as it is now, but yeah running everything always seems fine too. Let's go with that and we can always add the `--browsers chromium` flag if we end up hating it!